external/storm-kafka/src/jvm/storm/kafka/PartitionManager.java

```diff
@@ -252,28 +253,26 @@ public void fail(Long offset) {
     public void commit() {
         long lastCompletedOffset = lastCompletedOffset();
         if (_committedTo != lastCompletedOffset) {
-            LOG.debug("Writing last completed offset (" + lastCompletedOffset + ") to ZK for " + _partition + " for topology: " + _topologyInstanceId);
+            LOG.debug("Writing last completed offset (" + lastCompletedOffset + ") for " + _partition + " for topology: " + _topologyInstanceId);
             Map<Object, Object> data = (Map<Object, Object>) ImmutableMap.builder()
-                    .put("topology", ImmutableMap.of("id", _topologyInstanceId,
+                    .put("topology", ImmutableMap.of(
```

Comment:
the auto formatting could be improved :(  Probably not worth your time to do this as I face the same issue, and fall back to my own formatting.